scripts/gen-api.ts

```diff
@@ -312,7 +312,8 @@ const mapMember =
         params,
         returns,
         sees,
-        categories
+        categories,
+        dir: ['head', 'init', 'last', 'tail'].includes(name) ? 'common' : 'src'
```

Comment:
It'd be better to replace this hardcoded list with dynamic paths, but I didn't see paths in `temp/fonction.api.json`.

Reply:
LGTM
Thanks for your contribute!